includes/admin.form.inc

```diff
@@ -263,21 +263,49 @@ function islandora_openseadragon_admin($form, &$form_state) {
  * Validates that the supplied djatoka base URL is a working djatoka install.
  */
 function djatoka_base_url_validate($element, &$form_state, $form) {
-  $url = $element['#value'];
+  $jp2_image = drupal_get_path('module', 'islandora_openseadragon') . '/test/test.jp2';
+  $image_url = file_create_url($jp2_image);
+  $server_url = $element['#value'];
+  $dimensions = '0,0,100,100';
```

Comment:
Just to DRY things out a bit, could use a variable to stash the magnitude for which we're testing... Would then be able to use it below, when actually testing... Dunno. May not be worth it? As in:
```php
$dimension = 100;
$dimensions = "0,0,$dimension,$dimension";
[... skip to below ...]
$dimension_test = $info['width'] == $dimension && $info['height'] == $dimension;
```
... shrug?